packages/volto/news/5530.bugfix

```diff
@@ -0,0 +1 @@
+Do not pin the `eslint-plugin-jsx-a11y` version in Volto. Fixed generator ESlint config. Added tests and docs. @sneridagh
```

Comment:
Same as above.

````suggestion
After creating a project using the project generator, running ESLint might return an error message in your console:

 ```console
✖ pnpm eslint --max-warnings=0 --fix:
ESLint found too many warnings (maximum: 0).
```

- Fixed the project generator's ESLint configuration.
- Added code quality checks to the CI to ensure a generated project can run these checks.
- Added documentation for how to reconfigure ESLint in existing projects.

@sneridagh
````

Reply:
@stevepiercy this is a changelog, not an issue tracker.

Reply:
Revised.